doc/reference/modules/basic_mapping.xml

```diff
@@ -2750,15 +2789,22 @@
                         <row>
                             <entry><literal>Timestamp</literal></entry>
                             <entry><literal>System.DateTime</literal></entry>
-                            <entry><literal>DbType.DateTime</literal> - as specific as database supports.</entry>
```

Comment:
> as specific as database supports

This was a lie. It just does not cut milliseconds as the old `DateTime` was doing. But by example with Oracle, it will use a `timestamp(4)` while Oracle supports `timestamp(9)` (at least since 10g).